common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -45,6 +50,15 @@ class VerticalBlock(SequenceFields, XModuleFields, StudioEditableBlock, XmlParse
 
     show_in_read_only_mode = True
 
+    is_discussable = Boolean(
```

Comment:
In line with conversation elsewhere, I'm inclined to hold back on logic related to in-line discussions for the initial release(s).

Until we have a UI to selectively disable units, every unit will either be enabled or disabled (together, with the group), so we can punt on the complication for now (the recursive logic, should graded assignments have discussions auto-enabled during recursion, etc.?).